manticore/core/workspace.py

```diff
@@ -482,12 +487,12 @@ def save_testcase(self, state, prefix, message=''):
         :param str message: The message to add to output
         :return: A state id representing the saved state
         """
-
         self._named_key_prefix = prefix
-        self._increment_id()
```

Comment:
NOTE: This has been removed because we already increment the id when we create a `Testcase` instance.

This `Testcase` instance is not used here *yet*. It shall be, in the end.